src/main.ts

```diff
@@ -64,13 +68,21 @@ try {
 try {
   cweXml = new DOMParser().parseFromString(readFileSync(cweFilePath, 'utf8'))
 } catch (err) {
+  console.log(`Unable to load CWE file: ${err}`)
```

Comment:
`core.warning` or `core.info`

Reply:
There is already a message passed as part of core.setFailed()